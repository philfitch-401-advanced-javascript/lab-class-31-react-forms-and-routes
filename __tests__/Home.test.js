import React from 'react';
import { shallow } from 'enzyme';
import Home from '../src/components/Home';

describe('Home Component', () => {
  it('renders Home', () => {
    const wrapper = shallow(<Home />);
    expect(wrapper).toMatchSnapshot();
  });
  
});
