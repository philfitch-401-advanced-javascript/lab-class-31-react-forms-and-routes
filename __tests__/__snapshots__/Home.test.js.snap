// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component renders Home 1`] = `
<h1>
  Home
</h1>
`;
